swap_meet/vendor.py

```diff
@@ -1,2 +1,92 @@
+""" Initiation of class Vendor """
+
 class Vendor:
-    pass
\ No newline at end of file
+		""" Represents a unique entity from which one can trade items. """
+		def __init__(self, inventory=None):
+				""" Initiate class vendor with empty inventory. """
+				self.inventory = [] if inventory is None else inventory
+
+		def add (self, item):
+			""" Adds an item to inventory and returns the added item """
+			self.inventory.append(item)
+			return item
```

Comment:
Consider adding in a guard clause, aka a condition to check whether or not the `item` being appended is, in fact, an item.

This doesn't affect the tests nor was it required, but it never hurts to validate input when we can!